packages/clay-core/src/focus-trap/FocusTrap.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * SPDX-FileCopyrightText: © 2021 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {FOCUSABLE_ELEMENTS, FocusScope} from '@clayui/shared';
+import React, {useEffect, useRef} from 'react';
+
+type Props<T> = {
```

Comment:
Just noticed that the generic type is not being used, maybe you don't need it anymore now.